src/expressions/CMakeLists.txt

```
@@ -98,6 +99,7 @@ target_link_libraries(expressions
         linear-problem-api
         PRIVATE
         Antares::antares-study-system-model
+        Antares::antares-study-system-model
```

oops sorry, actually my last suggestion duplicated the line
```suggestion
        PRIVATE
        Antares::antares-study-system-model
```